package/src/i18n/en.json

```diff
@@ -68,6 +68,7 @@
   "Only visible to you": "Only visible to you",
   "Open Settings": "Open Settings",
   "Option": "Option",
+  "Option already exists": "Option already exists",
```

Comment:
Again, why are we doing such breaking changes ? It's for V8, sure - but what's the reason behind it ? These texts were added in line with the designs we had back then, so unless something changed there I don't see the point in doing this.

Reply:
This is respecting the error returned from the LLC

Reply:
Not sure what that means or why it implies a breaking change in our translations

Reply:
We are directly respecting the error returned from the LLC(https://github.com/GetStream/stream-chat-js/blob/master/src/messageComposer/middleware/pollComposer/state.ts#L49). Yes, the text is changed but we don't have a way here.

Reply:
Well, the non-breaking way to do this is to do something like:

```
"This is already an option": "Option already exists",
```
or something of the sort but if you say we need it in order to react to errors better then it's fine. Please also mention it in the docs when you write them